docs/react-integration.md

```diff
@@ -129,35 +124,33 @@ ReactDOM.render(
 )
 ```
 
-需要注意的是我们并不推荐每一个不同的 `值（value）` 都通过不同的 `Provider`来传递 . 在使用Mobx的过程中不需要这样做, 因为共享的可观察对象会更新他自己。
```

Comment:
Note that we don't recommend ever replacing the value of a Provider with a different one. Using MobX, there should be no need for that, since the observable that is shared can be updated itself.

Reply:
感觉原来的中文翻译和英文原文的意思不同。